src/main/kotlin/step3/Race.kt

```diff
@@ -0,0 +1,34 @@
+package step3
+
+import java.util.Random
+
+object Race {
+    private const val MIN_VALUE_TO_MOVE = 4
+
+    fun initiate(carNumber: Int): ArrayList<Car> {
+        val carList = ArrayList<Car>()
+        for (i in 1..carNumber) {
+            carList.add(Car(0))
+        }
+        return carList
+    }
+
+    fun makeTurn(cars: ArrayList<Car>): ArrayList<Car> {
+        for (car in cars) {
+            car.distance = getDistanceForEachCar(car.distance)
+        }
+        return cars
+    }
+
+    fun getDistanceForEachCar(distance: Int): Int {
+        return if (successToMove(getRandomNumber())) distance + 1 else distance
+    }
+
+    fun successToMove(random: Int): Boolean {
+        return random >= MIN_VALUE_TO_MOVE
+    }
+
+    fun getRandomNumber(): Int {
+        return Random().nextInt(10)
```

Comment:
코틀린의 `Random`을 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        return Random.nextInt(10)
```

Reply:
자바의 Random 대신 코틀린의 Random 함수를 import 하여 사용하도록 수정하였습니다.